modules/cluster-logging-logstore-tolerations.adoc

```diff
@@ -0,0 +1,116 @@
+
+.Elasticsearch log store pods default tolerations
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: elasticsearch-example
+  namespace: openshift-logging
+spec:
+# ...
+  tolerations:
+  - effect: NoSchedule
+    key: node.kubernetes.io/disk-pressure
+    operator: Exists
+  - effect: NoExecute
+    key: node.kubernetes.io/not-ready
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoExecute
+    key: node.kubernetes.io/unreachable
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoSchedule
+    key: node.kubernetes.io/memory-pressure
+    operator: Exists
+# ...
+----
+
+.LokiStack log store pods default tolerations
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: lokistack-example
+  namespace: openshift-logging
+spec:
+# ...
+  tolerations:
+  - effect: NoExecute
+    key: node.kubernetes.io/not-ready
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoExecute
+    key: node.kubernetes.io/unreachable
+    operator: Exists
+    tolerationSeconds: 300
+  - effect: NoSchedule
+    key: node.kubernetes.io/memory-pressure
+    operator: Exists
+# ...
+----
+
+You can configure a toleration for log store pods by adding a taint and then modifying the `tolerations` syntax in the `ClusterLogging` custom resource (CR).
+
+.Prerequisites
+
+* You have installed the {clo}.
+* You have installed the {oc-first}.
+* You have deployed an internal log store that is either Elasticsearch or LokiStack.
+
+.Procedure
+
+. Add a taint to a node where you want to schedule the {logging} pods, by running the following command:
++
+[source,terminal]
+----
+$ oc adm taint nodes <node_name> <key>=<value>:<effect>
+----
++
+.Example command
+[source,terminal]
+----
+$ oc adm taint nodes node1 lokistack=node:NoExecute
+----
++
+This example places a taint on `node1` that has key `lokistack`, value `node`, and taint effect `NoExecute`. Nodes with the `NoExecute` effect schedule only pods that match the taint and remove existing pods that do not match.
+
+. Edit the `logstore` section of the `ClusterLogging` CR to configure a toleration for the log store pods:
++
+.Example `ClusterLogging` CR
+[source,yaml]
+----
+apiVersion: logging.openshift.io/v1
+kind: ClusterLogging
+metadata:
+# ...
+spec:
+# ...
+  logStore:
+    type: lokistack
+    elasticsearch:
+      nodeCount: 1
+      tolerations:
+      - key: lokistack # <1>
+        operator: Exists # <2>
+        effect: NoExecute # <3>
+        tolerationSeconds: 6000 # <4>
+# ...
+----
+<1> Specify the key that you added to the node.
+<2> Specify the `Exists` operator to require a taint with the key `lokistack` to be present on the node.
+<3> Specify the `NoExecute` effect.
+<4> Optional: Specify the `tolerationSeconds` parameter to set how long a pod can remain bound to a node before being evicted.
+
+This toleration matches the taint created by the `oc adm taint` command. A pod with this toleration can be scheduled onto `node1`.
```

Comment:
Maybe attach this to the step before it? Would require use of `--` open blocks around the codeblock+callouts, I believe.

Reply:
This is how all of the similar procedures were laid out in the existing content, so I'll leave it alone for now 😉